clients/cobol-lsp-vscode-extension/esbuild.js

```diff
@@ -0,0 +1,100 @@
+const esbuild = require('esbuild');
+const glob = require('glob');
+const path = require('path');
+const polyfill = require('@esbuild-plugins/node-globals-polyfill');
+
+const production = process.argv.includes('--production');
+const watch = process.argv.includes('--watch');
+
+/**
+ * This plugin hooks into the build process to print errors in a format that the problem matcher in
+ * Visual Studio Code can understand.
+ * @type {import('esbuild').Plugin}
+ */
+const esbuildProblemMatcherPlugin = {
+    name: 'esbuild-problem-matcher',
+
+    setup(build) {
+        build.onStart(() => {
+            console.log('[watch] build started');
+        });
+        build.onEnd((result) => {
+            result.errors.forEach(({ text, location }) => {
+                console.error(`✘ [ERROR] ${text}`);
+                console.error(`    ${location.file}:${location.line}:${location.column}:`);
+            });
+            console.log('[watch] build finished');
+        });
+    },
+};
+
+
+/**
+ * For web extension, all tests, including the test runner, need to be bundled into
+ * a single module that has a exported `run` function .
+ * This plugin bundles implements a virtual file extensionTests.ts that bundles all these together.
+ * @type {import('esbuild').Plugin}
+ */
+const testBundlePlugin = {
+    name: 'testBundlePlugin',
+    setup(build) {
+        build.onResolve({ filter: /[\/\\]extensionTests\.ts$/ }, args => {
+            if (args.kind === 'entry-point') {
+                return { path: path.resolve(args.path) };
+            }
+        });
+        build.onLoad({ filter: /[\/\\]extensionTests\.ts$/ }, async args => {
+            const testsRoot = path.join(__dirname, 'src/web/test/suite');
+            const files = await glob.glob('*.test.{ts,tsx}', { cwd: testsRoot, posix: true });
+            return {
+                contents:
+                    `export { run } from './mochaTestRunner.ts';` +
+                    files.map(f => `import('./${f}');`).join(''),
+                watchDirs: files.map(f => path.dirname(path.resolve(testsRoot, f))),
+                watchFiles: files.map(f => path.resolve(testsRoot, f))
+            };
+        });
+    }
+};
+
+async function main() {
+    const ctx = await esbuild.context({
+        entryPoints: [
+            'src/web/extension.ts',
+            'src/web/test/suite/extensionTests.ts'
+        ],
+        bundle: true,
+        format: 'cjs',
+        minify: production,
+        sourcemap: !production,
+        sourcesContent: false,
+        platform: 'browser',
+        outdir: 'dist/web',
+        external: ['vscode'],
+        logLevel: 'silent',
+        // Node.js global to browser globalThis
+        define: {
+            global: 'globalThis',
+        },
+
+        plugins: [
```

Comment:
It is because the package was removed from the DevDependencies in this earlier commit: https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2323/commits/c20d3181b8b15e0210b415758d8b5e90d129fca5
Is able to build once added back in. Fixed in latest commit.